jira/resilientsession.py

```diff
@@ -57,12 +57,6 @@ def raise_on_error(r, verb='???', **kwargs):
     # for debugging weird errors on CI
     if r.status_code not in [200, 201, 202, 204]:
         raise JIRAError(r.status_code, request=request, response=r, **kwargs)
-    # testing for the WTH bug exposed on
```

Comment:
Why are you removing this? This is a workaround for https://jira.atlassian.com/browse/JRASERVER-41559 which was never fixed.

Reply:
@ssbarnea perhaps I misunderstood the code that I removed. It looked to me like it wasn't actually doing anything.

Reply:
Remove any change that is unrelated to this feature and I will merge it as it seems to have its own tests which are passing.

Reply:
@ssbarnea will do. I'll break the other changes out into individual prs.